Makefile

```diff
@@ -455,14 +457,18 @@ npm-update: node-check node_modules
 	npm install --package-lock
 
 .PHONY: js
-js: node-check $(JS_DEST)
+js: node-check fomantic $(JS_DEST)
 
 $(JS_DEST): node_modules $(JS_SOURCES)
 	npx eslint web_src/js webpack.config.js
 	npx webpack
 
+.PHONY: fomantic
+fomantic: node-check node_modules
+	cd web_src/fomantic && npx gulp build
+
```

Comment:
Maybe it is possible to have this as a non-phony target that depends on `package-lock.json`, so it would only run if dependencies change.

Reply:
Also, I'd suggest the target to be adjusted to provide one of the output files, like `public/fomantic/something.js`.